app/index.js

```diff
@@ -10,8 +10,17 @@ import { App } from './features/app';
  */
 import '@atlaskit/css-reset';
 
+import { Provider } from 'react-redux';
+import { store } from './features/redux';
+
 /**
  * Render the main / root application.
- * $FlowFixMe
  */
-render(<App />, document.getElementById('app'));
+render(
+    <Provider store={store}>
+        <App />
+    </Provider>,
```

Comment:
Can you create a component in this very file, call it `Root`, which encapsulates this provider and app?

Reply:
Sure.